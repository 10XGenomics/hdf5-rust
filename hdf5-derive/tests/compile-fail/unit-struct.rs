use hdf5_metno_derive::H5Type;

#[derive(H5Type)]
//~^ ERROR proc-macro derive
//~^^ HELP Cannot derive H5Type for unit structs
struct Foo;

fn main() {}
