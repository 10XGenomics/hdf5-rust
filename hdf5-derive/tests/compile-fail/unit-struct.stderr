error: cannot derive `H5Type` for unit structs
 --> $DIR/unit-struct.rs:7:8
  |
7 | struct Foo;
  |        ^^^
